.travis.yml

```diff
@@ -1,12 +1,11 @@
 language: python
 python:
-- 2.7
 - 3.5
+- 3.8
```

Comment:
This was done in the other open PR. Please remove this change so we can avoid conflicts